client/src/components/pages/EditProfile.js

```diff
@@ -1,81 +1,164 @@
-import React, { useEffect, useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Container, Form, Button, Row, Col } from "react-bootstrap";
-import { useAuth } from "../../context/auth/AuthState";
+import { deleteUser, updateUser, useAuth, clearErrors } from "../../context/auth/AuthState";
+import AlertEditProfile from '../Alerts/AlertEditProfile';
+import AlertModal from "../Alerts/AlertModal";
+import { LOGOUT, UPDATE_USER_SUCCESS, UPDATE_USER_FAIL } from "../../context/types";
 
 function EditProfile() {
-	const [authState] = useAuth();
-	const { isAuthenticated, user } = authState;
-	const [validated, setValidated] = useState(false);
+	const [authState, authDispatch] = useAuth();
+	const { isAuthenticated, user, error} = authState;
+	const [alert, setShowAlert] = useState(false);
 
+	const [userData, setUserData] = useState({
+		firstName: '',
+		lastName: '',
+		organization: '',
+		email: ''
+	});
 
-	const handleSubmit = (event) => {
-		const form = event.currentTarget;
-		if (form.checkValidity() === false) {
-			event.preventDefault();
-			event.stopPropagation();
+	useEffect(() => {
+		setUserData({
+			firstName: isAuthenticated && user && user.firstName,
+			lastName: isAuthenticated && user && user.lastName,
+			organization: isAuthenticated && user && user.organization,
+			email: isAuthenticated && user && user.email,
+		});
+	}, [isAuthenticated, user, authDispatch]);
+
+
+	useEffect(() => {
+		if (UPDATE_USER_FAIL) {
```

Comment:
issue # 2 here. I tried to use {error} but it doesn't seem to print anything

Reply:
I see the issue with the errors. You don't want to pull the error type directly as it does contain anything by itself. You just want to call if(error) and the error will display the first one from the array of errors that are generated from express-validator. I will make this correction.